src/pages/configurePatch/configurePatchCore/ConfigureTasks.tsx

```diff
@@ -82,7 +87,15 @@ export const ConfigureTasks: React.FC<Props> = ({
   return (
     <TabContentWrapper>
       <Actions>
-        <Button variant={Variant.Primary}>Schedule</Button>
+        <Button
+          data-cy="schedule-patch"
+          variant={Variant.Primary}
+          onClick={onClickSchedule}
+          disabled={isEmpty(selectedVariantTasks) || loading}
+        >
+          {loading && <StyledIcon type="loading" />}
+          Schedule
+        </Button>
```

Comment:
I saw this interaction in the thread on slack can we generalize this and move it to its own component a button with a loading state would be super useful

Reply:
yes!

Reply:
Great ill wait for that so i can use it in my pr